internal/driver/driver_test.go

```diff
@@ -33,6 +36,81 @@ func TestDriver(t *testing.T) {
 	require.NoError(t, err)
 	t.Cleanup(func() { require.NoError(t, c.Close()) })
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/4146
-	_ = c
+	t.Run("Insert", func(t *testing.T) {
+		doc := must.NotFail(bson.NewDocument(
+			"insert", "values",
+			"documents", must.NotFail(bson.NewArray(
+				must.NotFail(bson.NewDocument("v", int32(1), "_id", bson.ObjectID([]byte("65f83bddef2048e47170b641")))),
+				must.NotFail(bson.NewDocument("v", int32(2), "_id", bson.ObjectID([]byte("65f83bddef2048e47170b642")))),
+			)),
+			"ordered", true,
+			"lsid", int32(0),
+			"txnNumber", int64(1),
+			"$db", "test",
+		))
+
+		section, err := must.NotFail(bson.NewDocument(
+			"Kind", int32(0),
+			"Document", doc,
+		)).Encode()
+
+		require.NoError(t, err)
+
+		body, err := wire.NewOpMsg(section)
+		require.NoError(t, err)
+
+		msgBin, err := body.MarshalBinary()
+		require.NoError(t, err)
+
+		// TODO verify header
+		header := wire.MsgHeader{
+			MessageLength: int32(len(msgBin) + wire.MsgHeaderLen),
+			RequestID:     13,
+			OpCode:        wire.OpCodeMsg,
+		}
```

Comment:
- Should we verify headers and bodies of every step here, or just check if documents are present in getMore calls?
- If we should, are we going to implement similar checks in future tests later? In that case I'll create helper function for that

Reply:
I think we should do the full check for Insert, Find, and GetMore. But then I'm not sure what other commands or tests we would need.

Overall, that should not be an integration test of FerretDB or MongoDB